.tx/config

```diff
@@ -1,5 +1,6 @@
 [main]
 host = https://www.transifex.com
+lang_map = zh-Hant: zh_TW
```

Comment:
@suricactus can you create a separate PR for this one so we get back to green and can wait for a review by @signedav and/or @nirvn of this one?

Reply:
You were way too fast :)